src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.body {
```

Comment:
The selector '.body' should be 'body' without the dot, as 'body' is an HTML element, not a class.